### PR TITLE
[DEV-1055] Add webinars list page

### PR DESCRIPTION
**Pull request by marcobottaro**
#### List of Changes

Add webinars list page

#### Screenshots (if appropriate):
image

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by marcobottaro** [comment]
Remember to update SiteHeader menu!

**Review by MarcoPonchia — CHANGES_REQUESTED** [review]

**Review by MarcoPonchia — APPROVED** [review]

File: apps/nextjs-website/src/components/organisms/WebinarsTemplate/WebinarsTemplate.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  const getPastWebinars = (currentWebinars: readonly Webinar[]) =>
+    currentWebinars
+      .filter(
+        ({ endDateTime, startDateTime }) =>
+          endDateTime &&
+          startDateTime &&
+          new Date(endDateTime).getTime() <= new Date().getTime()
+      )
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      .sort((a, b) => (a.startDateTime! < b.startDateTime! ? 1 : -1));
```
Comment (tommaso1) [inline apps/nextjs-website/src/components/organisms/WebinarsTemplate/WebinarsTemplate.tsx:40]: I'd move these 2 functions outside the component, or wrap them in a `useCallback` hook so they are not redefined every re-render of the component

**Review by tommaso1 — APPROVED** [review]

File: apps/nextjs-website/src/components/organisms/WebinarsTemplate/WebinarsTemplate.tsx (hunk 1 above)
Comment (marcobottaro) [reply apps/nextjs-website/src/components/organisms/WebinarsTemplate/WebinarsTemplate.tsx:40]: When using `useCallback` the current date never changes. I think it's better to leave the code as it is

Comment (marcobottaro) [reply apps/nextjs-website/src/components/organisms/WebinarsTemplate/WebinarsTemplate.tsx]: Done

**Review by datalek — APPROVED** [review]

**Pull request merged**